modules/tiles/src/tileset/helpers/i3s-lod.ts

```diff
@@ -100,24 +99,14 @@ export function getI3ScreenSize(tile, frameState) {
   if (d <= 0.0) {
     return 0.5 * fltMax;
   }
-  let screenSizeFactor = calculateScreenSizeFactor(tile, frameState);
-  // viewport changed in deck.gl v8.0
-  screenSizeFactor *= mbsRNormalized / Math.sqrt(d) / viewport.scale;
+  // Hack: 300 is a Magic number to get the correct LoD
```

Comment:
Nit: add your PR comment with stack overflow link etc here.

PS- We REALLY need test cases for traversal, could you add at least one minimal test?

Reply:
Done